modules/function/src/main/java/com/opengamma/strata/function/marketdata/curve/CurveGroupMarketDataFunction.java

```diff
@@ -31,44 +32,42 @@
 import com.opengamma.strata.market.curve.NodalCurveDefinition;
 import com.opengamma.strata.market.id.CurveGroupId;
 import com.opengamma.strata.market.id.CurveInputsId;
-import com.opengamma.strata.pricer.calibration.CalibrationMeasures;
 import com.opengamma.strata.pricer.calibration.CurveCalibrator;
 import com.opengamma.strata.pricer.rate.ImmutableRatesProvider;
 
 /**
- * Market data function that builds a {@link CurveGroup}.
+ * Market data function that builds a curve group.
+ * <p>
+ * This function calibrates curves, turning a {@link CurveGroupDefinition} into a {@link CurveGroup}.
  */
 public class CurveGroupMarketDataFunction implements MarketDataFunction<CurveGroup, CurveGroupId> {
 
-  /** The analytics object that performs the curve calibration. */
+  /**
+   * The default analytics object that performs the curve calibration.
+   */
   private final CurveCalibrator curveCalibrator;
```

Comment:
I'm wondering if having the curve calibrator here is confusing. Would it be cleaner if the calibrator were always built using the root finder config from the `MarketDataConfig`. The logic to use the default root finder config would go in `MarketDataConfig` or `MarketDataConfigBuilder`.